packages/module/patternfly-docs/content/extensions/chatbot/examples/Messages/UserMessage.tsx

```diff
@@ -2,25 +2,128 @@ import React from 'react';
 
 import Message from '@patternfly/chatbot/dist/dynamic/Message';
 import userAvatar from './user_avatar.svg';
+import { FormGroup, Radio } from '@patternfly/react-core';
 
 export const UserMessageExample: React.FunctionComponent = () => {
-  const markdown = `A paragraph with *emphasis* and **strong importance**.
+  const [variant, setVariant] = React.useState('code');
 
-> A block quote with ~strikethrough~ and a URL: https://reactjs.org.
+  /* eslint-disable indent */
+  const renderContent = () => {
+    switch (variant) {
+      case 'code':
+        return code;
+      case 'inlineCode':
+        return inlineCode;
+      case 'heading':
+        return heading;
+      case 'emphasis':
+        return emphasis;
+      case 'blockQuotes':
+        return blockQuotes;
+      case 'orderedList':
+        return orderedList;
+      case 'unorderedList':
+        return unorderedList;
+      case 'moreComplexList':
+        return moreComplexList;
+      case 'link':
+        return link;
+      default:
+        return;
+    }
+  };
+  /* eslint-enable indent */
 
-Here is an inline code - \`() => void\`
+  const code = `
+Here is some YAML code:
 
-Here is an ordered list: 
+~~~yaml
+apiVersion: helm.openshift.io/v1beta1/
+kind: HelmChartRepository
+metadata:
+  name: azure-sample-repo0oooo00ooo
+spec:
+  connectionConfig:
+  url: https://raw.githubusercontent.com/Azure-Samples/helm-charts/master/docs
+~~~
 
-1. Item 1
-2. Item 3
-3. Item 4
+Here is some JavaScript code:
 
-Here is an unordered list:
+~~~js
+import React from 'react';
+
+const MessageLoading = () => (
+  <div className="pf-chatbot__message-loading">
+    <span className="pf-chatbot__message-loading-dots">
+      <span className="pf-v6-screen-reader">Loading message</span>
+    </span>
+  </div>
+);
+
+export default MessageLoading;
+
+~~~
+`;
+
+  const heading = `
+# h1 Heading
```

Comment:
The heading examples are using black text instead of white for me: 
<img width="317" alt="image" src="https://github.com/user-attachments/assets/e21260bf-78f5-43d1-b26b-fcb1f993081d" />
just in the user messages!

Reply:
Good call-out; thank you! I am stretched a little thin at the moment. Making mistakes. :)